tests/unit/components/sl-button-test.js

```diff
@@ -172,3 +150,6 @@ test( 'Button supports click event bubbling', function( assert ) {
         'Button click event will not propagate when bubbles is false'
     );
 });
+
+// This test will need to have the ember-stream/streamService available to test
+skip( 'showModalWithStreamName property triggers modal to open', function() {} );
```

Comment:
I know we previously talked about this and decided to skip this test but now that I'm thinking about things even if we knew how to instantiate and use the `ember-stream/streamService` it would still not be the correct approach in the test to use it.  Conceptually, this may just as well be a database, API or some other service.  The point being, we do not want to test any of it, but rather our interaction with its expected existence.  We should be mocking this service for our tests.  Now the mocking and registering it for usage may still require us to skip the test for now, but if so then we should refine the comment to indicate this rather than what it currently says.